README.md

```diff
@@ -34,7 +34,7 @@ memory-serve is designed to work with [axum](https://github.com/tokio-rs/axum)
 ## Usage
 
 Provide a relative path to the directory containing your static assets
-to the [`load_assets!`] macro. This macro creates a data structure intended to
+using the `ASSET_PATH` environment variable. This macro creates a data structure intended to
 be consumed by [`MemoryServe::new`]. Calling [`MemoryServe::into_router()`] on
 the resulting instance produces a axum
```

Comment:
Comment still mentions the removed macro.

Feel free to write something different but I pulled some from the description of the PR to maybe save you some time:
```suggestion
Provide a relative path to the directory containing your static assets
using the `ASSET_PATH` environment variable. This path will be used in a build script and any changes to files in that path or the variable itself will trigger a rerun and will load fresh assets. The output of the build script is consumed by [`MemoryServe::new`]. Calling [`MemoryServe::into_router()`] on
the resulting instance produces a axum
```

Reply:
Thanks for this! I took your suggestion and also mentioned the build script instead of the macro further on. See [7fad903](https://github.com/tweedegolf/memory-serve/pull/15/commits/7fad903403a87d7ae26ef3de8b5d535c454e2cd0)